doc/nrf/gs_programming.rst

```diff
@@ -86,8 +86,25 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
 
    .. build_SES_projimport_start
 
-4. Click :guilabel:`OK` to add the project to SES.
-   You can now work with the project in the IDE.
+4. Click :guilabel:`OK` to add the project to SES. If you are Importing the project for the first time, you need to request and activate a license.
```

Comment:
Actually, would it make sense to make the part about the license into a subsection? 
```suggestion

First-time project license activation
=========================

If you are adding the project to SES for the first time, you need to request and activate a license.
The following notification appears after you click :guilabel:`OK` to add the project to SES:

<figure>

To request and activate the license, complete the following steps:

<steps a through d>
```